app/src/main/java/com/spidev/android_arquitecture_component_demo/repository/model/entity/Person.java

```diff
@@ -0,0 +1,24 @@
+package com.spidev.android_arquitecture_component_demo.repository.model.entity;
+
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.PrimaryKey;
+
+/**
+ * Created by Bill on 26/12/2017.
+ */
+@Entity
+public class Person {
+    @PrimaryKey
+    public int idPerson;
+
+    @ColumnInfo(name = "full_name")
```

Comment:
Why have you used @ColumnInfo(name = "full_name") ??? it's necessary?

Reply:
se usa el @columninfo  si deseas llamar distinto a tu campo de tu tabla